README.md

```diff
@@ -180,11 +180,11 @@ an AlloyDB instance. The proxy supports the following options:
 3. [Application Default Credentials (ADC)][adc] if neither of the above have
    been set.
 
-Note: Any account connecting to AlloyDB instance will need one of the following
-IAM roles:
+Any account connecting to an AlloyDB instance will need the following
```

Comment:
I find the word "account" very vague and not a term we frequently use. May I suggest "IAM principal" as a replacement?

Also is Service Usage Consumer only need for auto IAM auth connections or all connections?

Reply:
Good call. I missed the weird wording. And yes, as of v1.5.0, all connections need these two roles.